Removed repetitiveness and put jQuery in node.html

BeepBot99 opened the pull request:
**What problem did you fix?**
I used the DRY principle (don't repeat yourself) by removing repetitiveness. I also put jQuery in for better readability.

**What type of pull request are you making?**
Code Refactor

**What changes were made?**
I refactored the code for the `addLine()` function into a `forEach` loop and I added jQuery and changed the code to reflect that.

**How was this tested?**
It works.

Mesure73L reviewed (changes requested):
Everything works as intended except for when you type a command, the `admin@AppleDeck $` should be red (this PR makes it white).

What it should be:
![image](https://github.com/user-attachments/assets/24ff8c5a-bb5a-49cf-8f35-d255ac352d28)

What it is now:
![image](https://github.com/user-attachments/assets/aa7d7302-2e31-4bb2-84ae-fc0e75ceff7c)

Additionally, if you could put jQuery in the Built With section in README.md that would be helpful, otherwise I can do it.

Thanks for your contributions!

BeepBot99 commented:
In the latest commit, I...
* Changed much of the styling to using flexboxes
* Changed IDs to classes in CSS
* Added a send button
* Added an `aria-label` to the button and `aria-live` to `#output`
* I changed the outer `div` to `main` for better accessibility
* I fixed the admin not showing up as red
